src/components/Board.tsx

```diff
@@ -0,0 +1,91 @@
+import { DragEvent, useState } from 'react';
+
+import Header from '@/components/board/header';
+import Form from '@/components/board/form';
+import Item from '@/components/board/item';
+
+import useBoolean from '@/hooks/useBoolean';
+
+import { container, list } from '@/components/board.css';
+import { ADD_TODO, DELETE_TODO, TodoType } from '@/hooks/useTodoStore';
+
+export type Todo = string[];
+
+type Props = {
+  type: TodoType;
+  todos: Todo;
+  dispatch: (action: any) => any;
+};
+
+export default function Board({ type, todos, dispatch }: Props) {
+  const { bool: showForm, setFalse, toggle } = useBoolean(false);
+  const [draggingType, setDraggingType] = useState('');
+  const [draggingTodo, setDraggingTodo] = useState('');
+
+  const todosCount = todos.length;
+
+  const handleDragStart = ({ type, text }: { type: string; text: string }) => {
+    setDraggingType(type);
+    setDraggingTodo(text);
+  };
+
+  const handleDragOver = (event: DragEvent<HTMLUListElement>) => {
+    event.preventDefault();
+
+    console.log('dragover');
+  };
+
+  const handleDrop = (event: DragEvent<HTMLUListElement>) => {
+    event.preventDefault();
+
+    dispatch({
+      type: DELETE_TODO,
```

Comment:
P4: DELETE_TODO가 어떤 constant 인지 명시적으로 드러나면 더 좋았을 것 같아요!
```suggestion
      type: ACTION_TYPE.DELETE_TODO,
```